types/tiny-slider-react/index.d.ts

```diff
@@ -252,6 +252,7 @@ export interface TinySliderProps {
     onTouchStart?: ((info: TinySliderInfo) => void) | undefined;
     onTouchMove?: ((info: TinySliderInfo) => void) | undefined;
     onTouchEnd?: ((info: TinySliderInfo) => void) | undefined;
+    children?: React.ReactNode;
```

Comment:
You could also use [PropsWithChildren](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/f0e47eff0f2cdfe7933d6990f02119de098f7279/types/react/index.d.ts#L799)